packages/graphile-build-pg/src/QueryBuilder.js

```diff
@@ -73,6 +73,7 @@ class QueryBuilder {
   context: GraphQLContext;
   rootValue: any; // eslint-disable-line flowtype/no-weak-types
   supportsJSONB: boolean;
+  isSingleFieldSelector: boolean;
```

Comment:
Lets make it clear this is internal:
```suggestion
  _isSingleFieldSelector: boolean;
```